bootloader/bootloadertest/bootloadertest.go

```diff
@@ -403,3 +413,30 @@ func (b *MockManagedAssetsBootloader) CommandLine(args []string) (string, error)
 	line := strings.Join(append([]string{b.StaticCommandLine}, args...), " ")
 	return strings.TrimSpace(line), nil
 }
+
+// MockManagedAssetsRecoveryAwreBootloader mocks a bootloader implementing the
```

Comment:
```suggestion
// MockManagedAssetsRecoveryAwareBootloader mocks a bootloader implementing the
```

Reply:
Thanks for spotting this. Fixed in #9004